index-pl.html

```diff
@@ -593,21 +593,21 @@ <h2>The Human Cost</h2>
       </label>
 
       <label class="row half bigsplash">
-        <div>Mask Effectiveness on Exhale: <span class="dotted">{{convertToPercent(eout)}}</span>%</div>
+        <div>Skuteczność maski przy Wydychaniu: <span class="dotted">{{convertToPercent(eout)}}</span>%</div>
         <slider v-model="eout" :min="0" :max="0.99" :step="0.01"></slider>
       </label>
 
       <label class="row half bigsplash">
-        <div>Mask Effectiveness on Inhale: <span class="dotted">{{convertToPercent(ein)}}</span>%</div>
+        <div>Skuteczność maski przy Wdychaniu: <span class="dotted">{{convertToPercent(ein)}}</span>%</div>
         <slider v-model="ein" :min="0" :max="0.99" :step="0.01"></slider>
       </label>
 
       <graph :traces="graph3Traces(
         '% Infected'
         )" :layout="graph3Layout(
-        'How Masks Reduce Infections',
-        'Percentage of People Who Wear Masks',
-        'Percentage of People Who Will Be Infected'
+        'Jak maski zmniejszają ilość zakażeń',
+        'Procent populacji noszący maski',
```

Comment:
Metonimia :) (komentarz by dostać lajka od @fenoloftaleina )

Reply:
Nie wiedziałem o co chodzi więc sprawdziłem wikipedię. Dalej nie wiem.

Reply:
Można uprościć to do zwrotu: "Procent nosi maski". "Procent" de facto nie nosi maski, ale wiadomo, że w tym zdaniu chodzi koniec końców o ludzi. #metonimia #zamiennia #czytamFranzena

Reply:
Ok, to bawcie się.